src/index.ts

```diff
@@ -251,9 +253,3 @@ export { UpdateTotp } from "./types/UpdateTotp";
 export { UpdateTotpResponse } from "./types/UpdateTotpResponse";
 export { VerifyEmail } from "./types/VerifyEmail";
 export { VoteView } from "./types/VoteView";
-export {
```

Comment:
This one needs to be added back in. Its annoying that my script removes it, and I wish there was another way around it.